src/router.js

```diff
@@ -0,0 +1,31 @@
+import app from 'ampersand-app'
+import Router from 'ampersand-router'
+import ModelPage from './pages/model'
+import HomePage from './pages/home'
+import React from 'react'
+import Layout from './layouts/application'
+
+export default Router.extend({
+  render (page, isFluid=false) {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'default parameters' is only available in ES6 (use esnext option).